backend/apis/parse-docket-pdf.utils.js

```diff
@@ -84,8 +84,16 @@ function parseCharges(lines, sections) {
   }
 
   walkSection(lines, sections, "CHARGES", addDispositionToCharges);
+  let chargesWithSeverity = charges.map(elm => {
```

Comment:
In general, a good rule of thumb is to start variables out as `const`s and then switch them to `let`s as needed. Also, renaming the `elm` variable makes the rest of the code easier to follow:

```suggestion
  const chargesWithSeverity = charges.map(charge => {
```

Reply:
Ok yeah that makes sense, I should have made them all const's and naming makes sense to make it easier to read. thanks